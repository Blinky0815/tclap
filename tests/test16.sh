#!/bin/sh

../examples/test3  -s one homer -B -Bh > tmp.out 2>&1

if cmp -s tmp.out test16.out; then
	exit 0
else 
	exit 1
fi

