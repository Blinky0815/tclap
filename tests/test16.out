PARSE ERROR: for argument:   -B (--existTest B)
             Argument already set!


USAGE: 

    ../examples/test3 [-i <int>] ...  -s <string> [-B] [--] [-v] [-h] <string> <fileNameString> ...

Where: 

   -i <int>,  --intTest <int>  (accepted multiple times)
     (value required)  multi int test

   -s <string>,  --stringTest <string>
     (required)  (value required)  string test

   -B,  --existTest B
     exist Test B

   --,  --ignore_rest
     Ignores the rest of the labeled arguments following this flag.

   -v,  --version
     Displays version information and exits.

   -h,  --help
     Displays usage information and exits.

   <string>
     (required)  (value required)  unlabeld test

   <fileNameString>  (accepted multiple times)
     (value required)  file names



this is a message

